src/stan/math/prim/scal/meta/VectorView.hpp

```diff
@@ -46,8 +46,8 @@ namespace stan {
 
     VectorView(std::vector<scalar_t>& v) : x_(&v[0]) { }
 
-    template <int R, int C>
-    VectorView(Eigen::Matrix<scalar_t,R,C>& m) : x_(&m(0)) { }
+    template <typename ET>
```

Comment:
what does ET mean? Why is this function templated as opposed to just using `scalar_t`?